06-best-practices/code/infrastructure/README.md

```diff
@@ -0,0 +1,71 @@
+## IaC w/ Terraform
+
+### Project infrastructure modules in GCP:
```

Comment:
AWS probably, not GCP

Reply:
```
06-best-practices/code/infrastructure/README.md
```
You can probably put it one level up? e.g. `06-best-practices/iac`

Reply:
Yes, will do, once the README is ready for iac.